packages/cysync-core/src/store/snackBar/index.ts

```diff
@@ -0,0 +1,53 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+import type { RootState } from '~/store';
+
+import { ISnackBarState } from './types';
+import { SnackBarProps } from '@cypherock/cysync-ui';
+import { sleep } from '@cypherock/cysync-utils';
+
+export * from './types';
+
+const initialState: ISnackBarState = {
+  isOpen: false,
+  props: undefined,
+  snackBarId: undefined,
+};
+
+export const openSnackBar = createAsyncThunk<any, SnackBarProps>(
+  'snackBar/openSnackBar',
+  async () => {
+    await sleep(5000);
+  },
```

Comment:
The timeout for snackbar should be configurable

Reply:
added timeout prop